src/Rap2hpoutre/LaravelLogViewer/LaravelLogViewer.php

```diff
@@ -81,20 +81,26 @@ public function setFile($file)
      */
     public function pathToLogFile($file)
     {
-        $logsPath = $this->storage_path;
-        $logsPath .= ($this->folder) ? '/' . $this->folder : '';
-
         if (app('files')->exists($file)) { // try the absolute path
             return $file;
         }
-
-        $file = $logsPath . '/' . $file;
-
-        // check if requested file is really in the logs directory
-        if (dirname($file) !== $logsPath) {
-            throw new \Exception('No such log file');
+        if (is_array($this->storage_path)) {
+            foreach ($this->storage_path as $value) {
```

Comment:
This will create a strange behavior. Assuming the user has two files with the same name: `dir1/laravel.log` and `dir2/laravel.log` (which is possible in multitenant app), the function will return... one of them (a "random one")... So it will be really bizarre for users.

Reply:
I will take a look over this :) excellent observation!

Reply:
Can this be solved by this issue I open https://github.com/rap2hpoutre/laravel-log-viewer/issues/171 but I was waiting on this PR be merge. 
 ??
"After approving PR #169 I will like to show the folder path in the display log name in that way the user can know from where the file is coming. I will create an issue just for tracking the feature with this same comment :)"